questionnaire.html

```diff
@@ -8,105 +9,108 @@
+                    alt="" srcset=""></span>
+            <p id="description">Please let us know how we can serve you.</p>
+            <hr>
+            <form id="survey-form">
+                <fieldset id="top">
+                    <legend>Personal Information</legend>
+                    <span class="form-group"><label id="name-label" for="name">Name: </label><input id="name"
+                            type="text" required placeholder="Enter you name"></span>
+                    <span class="form-group"><label id="name-label" for="name">Organization: </label><input id="orgName"
+                            type="text" required placeholder="Enter your Organization"></span>
+                    <span class="form-group"><label id="email-label" for="email">Email: </label><input id="email"
+                            type="email" min="0" max="130" placeholder="Enter your Email"></span>
+                </fieldset>
+                <fieldset>
+                    <legend>Tell Us A Little About Your Current Website</legend>
+                    <span class="form-group">
+                        <label id="comments-label">
+                            <ul>
+                                <li>What do you love about the current version of website, or not love?</li>
+                                <li>What do you find frustrating about your website, how or why is frustrating?</li>
+                                <li>What apects of your website diminish your user's experience or are frustrating to your site users?</li>
+                            </ul>
+                        </label>
+                        <textarea id="comment" placeholder="Enter your comment here..." rows="10"></textarea>
+                    </span>
+                </fieldset>
+                <fieldset>
+                    <legend>Let's Talk About The Future</legend>
+                    <span class="form-group">
+                        <label id="comments-label">
+                            <ul>
+                                <li>What needs or goals do you expect from your website that aren’t being
+                                    met by the current version?</li>
+                                <li>What is the purpose of your website, what should your website achieve?</li>
+                                <li>What function does your website serve your brand?</li>
+                                <li>What do you expect visitors accomplish when they visit your website?</li>
+                                <li>Who is the website's intended target audience, who are you speaking to?</li>
+                                <li>What image, look, or feel do you want your brand’s website to portray?</li>
+                            </ul>
+                        </label>
+                        <textarea id="comment0" placeholder="Enter your comment here..." rows="10" cols="50"></textarea>

```

Comment:
We do need to go back and fix the ids and labels for the textareas. Each one should have something descriptive.